README.md

```diff
@@ -47,9 +27,9 @@ const config = {
 };
 ```
 
-### Addon Configuration
+The addon configuration is done through Storybook's `preview`.
 
-The addon configuration is done through Storybook's `preview`. Few of the parameters are required for the addon to work properly:
+Few of the parameters are required for the addon to work properly:
```

Comment:
```suggestion
The following parameters are required for the addon to work properly:
```

Reply:
Not all the parameters which are written after this sentence are required (I'm detailing for each parameter which is required)
Maybe it should be a table

Reply:
Changed to a Table @talkor